src/harness/reference_models/montecarlo/monteCarloSimITM_test.py

```diff
@@ -0,0 +1,68 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
```

Comment:
This file is an example, so it should not be named  MAINSCRIPTNAME_test.py. This is reserved for the actual unit tests, so that automatic unit test run on all the modules can be run.
So the next file should be renamed with this name, and this one better rename as example_montecarloSimITM.py (or something equivalent)

Reply:
Good point. The file has been renamed